sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_client.py

```diff
@@ -41,7 +41,7 @@ class WebPubSubServiceClient(
     """
 
     def __init__(self, hub: str, endpoint: str, credential: "TokenCredential", **kwargs: Any) -> None:
-        _endpoint = "{endpoint}"
+        _endpoint = "{Endpoint}"
         self._config = WebPubSubServiceClientConfiguration(hub=hub, endpoint=endpoint, credential=credential, **kwargs)
```

Comment:
This file is generated with tool and will be overridden if regenerate, so please don't edit the generated code directly. And you can change code under _patch.py which will not be overridden .

Reply:
And I don't understand how this change could fix the issue. Please provide more analyze context in this PR.

Reply:
So the `_endpoint` is a placeholder to extract the endpoint from the connectionString (e.g: `Endpoint=https://chat-webpubsub.webpubsub.azure.com;AccessKey=x0ldy3zbkyZrGyQQq9OxXKNQL8NxBU7+v8XBdHBxPj0=;Version=1.0`). When the tool generated this file, it modifies the `Endpoint` to `endpoint`, which would break the code when calling `self._client.format_url(request.url, **path_format_arguments)`